### ISSUE TITLE
Make 'Add new Track' panel usable via Accessibility and define Navigation

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#8957]
### Description

Currently, the 'Add New Track' is not accessible via keyboard shortcuts. Let's make it and the subsequent buttons accessible via keyboard. 

It also requires navigation definition for the navigation tree. This is something for the developer to implement. 

![Image](https://github.com/user-attachments/assets/a58e3841-430d-43cc-bb76-ee7b5564733c)

**Comment by dozzzzer** [comment]
Tested

![Image](https://github.com/user-attachments/assets/00dfbbf6-bb9a-4e68-8f3f-f0cfa26c136a)

**Comment by dozzzzer** [comment]
Tested

**Comment by dozzzzer** [comment]
The buttons are accessible and visible in the navigation tree but aren't enabled: https://github.com/audacity/audacity/issues/9419
